docs/running-nio/in-the-cloud.md

```diff
@@ -30,26 +30,29 @@ A system is a group of interrelated elements. In the nio System Designer, a syst
 
   > **[info] <span class="allow-caps">Pubkeeper</span> server credentials**
   >
+  > <img class="right border" src="/img/cloud/editSystem.png" width="250" />
   > To access your **Pubkeeper** credentials:
-  > 1. Click on your system's name on the leftmost side of the System Designer.
-  > 1. Click the **edit** button in the contextual toolbar above the canvas.
+  > 1. Click on the **design** app icon in the app switcher on the left side of the System Designer.
+  > 1. Hover over the system card that you wish to get the Pubkeeper credentials of.
+  > 1. Click the **edit** button on the card.
```

Comment:
Excellent info box! What do you think about switching the order in step one so it is place -> action? "In the app switcher on the left side of the System Designer, click the **design** app icon." Step two ends with an awkward preposition, but I am not sure how best to fix: "Hover over the card of the system whose Pubkeeper credentials you need." ? Actually, I really like this sentence from above, "You can see the configuration of the cloud Pubkeeper server if you hover over a system card and click the **edit** icon in the toolbar." but does that fit here in these steps?

Reply:
I like it as the last step. Changing now.